pkg/pillar/cmd/zedrouter/networkinstance.go

```diff
@@ -527,10 +527,11 @@ func doNetworkInstanceCreate(ctx *zedrouterContext,
 			status.BridgeIPAddr, &status.NetworkInstanceConfig,
 			hostsDirpath, status.BridgeIPSets, status.Ipv4Eid)
 		startDnsmasq(bridgeName)
-
-		go DNSMonitor(bridgeName, bridgeNum)
 	}
 
+	// monitor the DNS and DHCP information
+	go DNSMonitor(bridgeName, bridgeNum, ctx, status)
```

Comment:
Don't we need a way to tell this go routine to exit when the bridge is deleted (due to the network instance being deleted)?

Reply:
In the bridge delete function, we have the 'DNSStopMonitor()', to send a signal to that go function, and it will exit from that function.

Reply:
OK